tgui/packages/tgui/interfaces/Flatpacker.tsx

```diff
@@ -188,7 +188,7 @@ const CostPreview = (props: CostPreviewProps) => {
               </TableCell>
               <Table.Cell verticalAlign="middle">
                 <div style={{ width: '200px' }}>
-                  {(material.amount / SHEET_MATERIAL_AMOUNT).toFixed(2)}
+                  x{(material.amount / SHEET_MATERIAL_AMOUNT).toFixed(2)}
```

Comment:
Is this intented?

Reply:
Yes it displays x8.61 sheets of iron instead of just 8.61 which looks more meaningful